src/app/filesort/show-file/show-file.component.ts

```diff
@@ -53,21 +73,81 @@ export class ShowFileComponent implements OnInit {
         typeFile:item.typeFile,
         newNameFile:""
       }
+      //All path status false trim *;
+      var tepmpath = this.returnPath(false,0,true);
       this.service.deleteFile(upItem).subscribe(data=>{
-        this.refreshFileSortList();
+        this.refreshFileSortList(this.actionChoose,tepmpath);
       })
     }
   }
-  refreshFileSortList()
+  openFolderClick(index:number)
+  {
+    var tepmpath = "";
+    if(index != 0)
+    {
+      tepmpath = this.returnPath(true,index,true);
+    }
+    this.refreshFileSortList(this.actionChoose,tepmpath);
+  }
+
+  checkClick(nameFolder:string)
+  {
+    var tempfields = this.returnPath(false,0,false);
+    console.log(tempfields + nameFolder);
+    this.refreshFileSortList(this.actionChoose,tempfields + nameFolder)
+  }
+  refreshFileSortList(val:string,some:string)
```

Comment:
it's filtering not sorting)

Reply:
private method?

Reply:
Please rename "some" variable to anothe one that can reflact the purpose of this variable